---
layout: default
markdown: kramdown
---

## About Me

I am a computational and statistical human geneticist at [Octant](https://www.octant.bio/), where I contribute to a range of R&D projects in high-throughput synthetic biology, drug discovery, and precision medicine along with some traditional biostatistics.

I specialize in the design, analysis, and interpretation of high-throughput multiplexed molecular measurements, particularly of functional human genetic variation. I obtained my PhD and MS in genetics and statistics (respectively) from Stanford University and my BS and BA in molecular biology and political science (respectively) from the University of Texas at Austin. My background includes practical experience and formal training in experimental molecular biology, bioinformatics/computational biology, and computational/applied statistics. In a past life, I also studied political theory/philosophy and contributed to research in comparative constitutional design.

## Scientific Interests

Computational and statistical genomics, functional and experimental human genomics, quantitative synthetic biology, gene regulation, high-throughput genetic measurement, biophysical/biochemical parameter inference, sequencing bioinformatics, applied machine learning in molecular biology, epigenomics and epitranscriptomics, small RNA biology, regularization and dimensionality reduction, allelic specificity of molecular phenotypes

## Education

|Ph.D.|Genetics|Stanford University School of Medicine|2021|
|M.S.|Statistics|Stanford University|2018|
|B.S.|Cell and Molecular Biology|University of Texas at Austin|2013|
|B.A.|Political Science, Philosophy, and Plan II Honors|University of Texas at Austin|2013|

## Publications
0. IN REVIEW: Schwartz CI<sup>&dagger;</sup>, **Abell NS<sup>&dagger;</sup>**, Li A, Aradhana, Tycko J, Truong A, Montgomery SB, Hess GT. [Towards optimizing diversifying base editors for high-throughput studies of single-nucleotide variants.](https://www.biorxiv.org/content/10.1101/2024.11.18.621003v1)
0. IN REVIEW: DeGorter MK, Goddard PC, Karakoc E, Kundu S, Yan SM, Nachun D, **Abell NS**, Aguirre M, Carstensen T, Chen Z, Durrant M, Dwaracherla VR, Feng K, Gloudemans MJ, Hunter N, Moorthy MPS, Pomilla C, Rodrigues KB, Smith CJ, Smith KS, Ungar RA, Balliu B, Fellay J, Flicek P, McLaren PJ, Henn B, McCoy RC, Sugden L, Kundaje A, Sandhu MS, Gurdasani D, Montgomery SB. [Transcriptomics and chromatin accessibility in multiple African population samples.](https://www.biorxiv.org/content/10.1101/2023.11.04.564839v1)
0. Howard CJ<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Osuna BA, Jones EM, Chan LY, Chan H, Artis DR, Asfaha JB, Bloom JS, Cooper AR, Liao A, Mahdavi E, Mohammed N, Su AL, Uribe GA, Kosuri S, Dickel DE<sup>&dagger;</sup>, Lubock NB<sup>&dagger;</sup>. [High resolution deep mutational scanning of the melanocortin-4 receptor enables target characterization for drug discovery.](https://pubmed.ncbi.nlm.nih.gov/40202051/). eLife. 2025:13:RP104725.
0. Debnath TK, **Abell NS<sup>&dagger;</sup>**, Li YR<sup>&dagger;</sup>, Devanathan SK, Navedo E, Xhemalce B. [NAT10 and N4-Acetylcytidine restrain R-loop levels and related inflammatory responses.](https://pubmed.ncbi.nlm.nih.gov/40138394/) Science Advances. 2025;11(13):eads6144.
0. Kellman LN, Neela PH, Srinivasan S. Siprashvilli Z, Shanderson RL, Hong AW, Rao D, Porter DF, Reynolds DL. Meyers RM, Guo MG, Yang X, Zhao Y, Wozniak GG, Donohue LKH, Shenoy R, Ko LA, Nguyen DT, Mondal S, Garcia OS, Elcavage L, Elfaki I, **Abell NS**, Tao S, Lopez CM, Montgomery SB, Khavari PA. [Functional analysis of cancer-associated germline risk variants.](https://pubmed.ncbi.nlm.nih.gov/39962238/) Nature Genetics. 2025;57(3):718-728.
0. Ipas H, Gouws E, **Abell NS**, Chiou P, Devanathan S, Herve S, Lee S, Mercado M, Reinsborough C, Halabelian L, Arrowsmith C, Xhemalce B. [ChemRAP uncovers specific mRNA translation regulation via RNA 5'-Pme.](https://pubmed.ncbi.nlm.nih.gov/38263329/) EMBO Reports. 2024;25(3):1570-1588.
0. Guo MG, Ang CE, Liu Y, Reynolds DL, Zhao Y, Donahue LKH, Yang X, Kellman LN, **Abell NS**, Yoo Y, Pershad Y, Geschwind DH, Montgomery SB, Duncan LE, Urban AE, Altman RB, Wernig M, Khavari PA. [Integrative analyses highlight functional regulatory variants associated with neuropsychiatric diseases.](https://pubmed.ncbi.nlm.nih.gov/37857935/) Nature Genetics. 2023;55(11):1876-1891.
0. Lu F<sup>&dagger;</sup>, Sossin A<sup>&dagger;</sup>, **Abell NS**, Montgomery SB, He Z. [Deep learning-assisted genome-wide characterization of massively parallel reporter assays.](https://pubmed.ncbi.nlm.nih.gov/36350674/) Nucleic Acids Research. 2023;11;50(20):11442-11454.
0. **Abell NS**, DeGorter MK, Gloudemans M, Greenwald E, Smith KS, He ZS, Montgomery SB. [Multiple causal variants underlie genetic associations in humans.](https://pubmed.ncbi.nlm.nih.gov/35298243/) Science. 2022;1247:eabj5117.
0. Reinsborough CW<sup>&dagger;</sup>, Ipas H<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Gouws EB, Williams JP, Mercado M, Van Den Berg C, Xhemalce B. [BCDIN3D RNA methyltransferase stimulates Aldolase C expression and glycolysis through let-7 microRNA in breast cancer cells.](https://pubmed.ncbi.nlm.nih.gov/33664453/) Oncogene. 2021;40(13):2395-2406.
0. McFadden MJ, McIntyre ABR<sup>&dagger;</sup>, Gokhale NS<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Ipas H, Xhemalce B, Mason CE, Horner SM. [Post-transcriptional regulation of antiviral gene expression by N6-methyladenosine.](https://pubmed.ncbi.nlm.nih.gov/33657363/) Cell Reports. 2021;34(9):108798.
0. Spector LP, Tiffany M, Ferraro NM, **Abell NS**, Montgomery SB, Kay MA. [Evaluating the genomic parameters for optimizing rAAV-mediated homologous recombination.](https://pubmed.ncbi.nlm.nih.gov/33248247/) Molecular Therapy. 2020;S1525-0016(20)30651-1.
0. Gay NR, Gloudemans M, Antonio ML, **Abell NS**, Balliu B, Park Y, Martin AR, Musharoff S, Rao A, Aguet F, Barbeira B, Bonazzola R, Hormozdiari F, GTEx Consortium, Ardlie K, Brown CD, Im HK, Lappalainen T, Wen X, Montgomery SB. [Impact of admixture and ancestry on eQTL analysis and GWAS colocalization in GTEx.](https://pubmed.ncbi.nlm.nih.gov/32912333/) Genome Biology. 2020;21(1):233.
0. Ferraro NM<sup>&dagger;</sup>, Strober BJ<sup>&dagger;</sup>, Einson J, **Abell NS**, Aguet F, Berveira AN, Bucan M, Castel S, Davis JR, Greenwald E, Hess GT, Hilliard AT, Kember RL, Kotis B, Park Y, Peloso G, Ramdas S, Scott AJ, Smail C, Tsang EK, Zekacat S, Ziosi M, Aradhana, TOPMed Lipids Working Group, Ardlie KG, Assimes TL, Bassik MC, Brown CD, Correa A, Hall I, Im HK, Li X, Natarajan P, GTEx Consortium, Lappalainen T, Mohammadi P, Montgomery SB, Battle A. [Transcriptomic signatures across human tissues identify functional rare genetic variation.](https://pubmed.ncbi.nlm.nih.gov/32913073/) Science. 2020;369:eaaz5900.
0. Balliu B, Durrant M, de Goede O, **Abell NS**, Li X, Liu B, Gloudemans MJ, Cook N, Smith KS, Pala M, Cucca F, Schlessinger D, Jaiswal S, Sabatti C, Lind L, Ingelsson E, Montgomery SB. [Genetic regulation of gene expression and splicing during a 10-year period of human aging.](https://pubmed.ncbi.nlm.nih.gov/31684996/) Genome Biology. 2019;20(1):230.
0. Reinsborough CW<sup>&dagger;</sup>, Ipas H<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Nottingham RM<sup>&dagger;</sup>, Yao J, Devanathan SK, Shelton SB, Lambowitz AM, Xhemalce B. [BCDIN3D regulates tRNA-His 3’ fragment processing.](https://pubmed.ncbi.nlm.nih.gov/31329584/) PLoS Genetics. 2019;15(7):e1008273.
0. Liu B<sup>&dagger;</sup>, Calton M<sup>&dagger;</sup>, **Abell NS**, Benchorin G, Gloudemans MJ, Hu J, Balliu B, Bok D, Montgomery SB, Vollrath D. [Genetic analyses of human fetal retinal pigment epithelium gene expression suggest ocular disease mechanisms.](https://pubmed.ncbi.nlm.nih.gov/31123710/) Communications Biology. 2019;2:186.
0. Shelton SB, Shah NM<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Devanathan S, Mercado M, Xhemalce B. [Crosstalk between the RNA methylation and histone binding activities of MePCE regulates P-TEFb activation on chromatin.](https://pubmed.ncbi.nlm.nih.gov/29425494/) Cell Reports. 2018;22(6):1374-1383.
0. **GTEx Consortium**. [Genetic effects on gene expression across human tissues.](https://pubmed.ncbi.nlm.nih.gov/29022597/) Nature. 2017;550(7675):204-213. [As member of the Consortium]
0. Zacharioudakis E<sup>&dagger;</sup>, Agarwal P<sup>&dagger;</sup>, Bartoli A, **Abell NS**, Kunalingam L, Bergoglio V, Xhemalce B, Miller KM, Rodriguez R. [Chromatin Regulates Genome Targeting with Cisplatin.](https://pubmed.ncbi.nlm.nih.gov/28474855/) Angewandte Chemie. 2017;56(23):6483-6487.
0. **Abell NS**<sup>&dagger;</sup>, Mercado M<sup>&dagger;</sup>, Cañeque T, Rodriguez R, Xhemalce B. [Click Quantitative Mass Spectrometry Identifies PIWIL3 as a Mechanistic Target of RNA Interference Activator Enoxacin in Cancer Cells.](https://pubmed.ncbi.nlm.nih.gov/28094937/) Journal of the American Chemical Society. 2017;139(4):1400-1403.
0. Tsang EK, **Abell NS**, Li X, Anaya V, Karczewski KJ, Knowles DA, Sierra RG, Smith KS, Montgomery SB. [Small RNA Sequencing in Cells and Exosomes Identifies eQTLs and 14q32 as a Region of Active Export.](https://pubmed.ncbi.nlm.nih.gov/27799337/) G3: Genes\|Genomes\|Genetics. (Bethesda). 2017;7(1):31-39.
0. Polioudakis D, **Abell NS**, Iyer VR. [MiR-191 Regulates Primary Human Fibroblast Proliferation and Directly Targets Multiple Oncogenes.](https://pubmed.ncbi.nlm.nih.gov/25992613/) PLoS ONE. 2015;10(5):e0126535.
0. Polioudakis D, **Abell NS**, Iyer VR. [miR-503 represses human cell proliferation and directly targets the oncogene DDHD2 by non-canonical target pairing.](https://pubmed.ncbi.nlm.nih.gov/25653011/) BMC Genomics. 2015;16:40.
0. Polioudakis D, Bhinge AA, Killion PJ, Lee BK, **Abell NS**, Iyer VR. [A Myc-microRNA network promotes exit from quiescence by suppressing the interferon response and cell-cycle arrest genes.](https://pubmed.ncbi.nlm.nih.gov/23303785/) Nucleic Acids Research. 2013;41(4):2239-54.

## Academic References

Dr. Stephen Montgomery - Stanford University School of Medicine

Dr. Blerta Xhemalce - Emory University (previously University of Texas at Austin)

Dr. Vishwanath Iyer - University of Texas at Austin





