---
layout: default
markdown: kramdown
---

## About Me

I am a Ph.D. Candidate in the Department of Genetics at Stanford University School of Medicine, and a member of the Montgomery Lab in the Departments of Pathology and Genetics. I also spend lots of time in the Statistics and Biomedical Data Science departments. I have experience and formal training in experimental molecular biology, bioinformatics/computational biology, and computational/applied statistics. In a past life, I also studied political theory/philosophy and researched comparative constitutional design.

## Research Interests

Computational and statistical genomics, functional and experimental human genomics, high-throughput genetic measurement, sequencing bioinformatics, applied machine learning in molecular biology, small RNA biology, regularization and dimensionality reduction, -omics integration, genome technology

## Education

|Ph.D.|Genetics|Stanford University School of Medicine|Present|
|M.S.|Statistics|Stanford University|2018|
|B.S.|Cell and Molecular Biology|University of Texas at Austin|2013|
|B.A.|Political Science, Philosophy, and Plan II Honors|University of Texas at Austin|2013|

## Publications

0. IN REVIEW: McFadden MJ, McIntyre ABR, Gokhale NS<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Ipas H, Mason CE, Xhemalce B, Horner SM. Post-transcriptional regulation of antiviral gene expression by N6-methyladenosine.
0. IN REVIEW: Spector LP, Tiffany M, Ferraro NM, **Abell NS**, Montgomery SB, Kay MA. Evaluating the genomic parameters for optimizing rAAV-mediated homologous recombination.
0. Ferraro NM<sup>&dagger;</sup>, Strober BJ<sup>&dagger;</sup>, Einson J, **Abell NS**, Aguet F, Berveira AN, Bucan M, Castel S, Davis JR, Greenwald E, Hess GT, Hilliard AT, Kember RL, Kotis B, Park Y, Peloso G, Ramdas S, Scott AJ, Smail C, Tsang EK, Zekacat S, Ziosi M, Aradhana, TOPMed Lipids Working Group, Ardlie KG, Assimes TL, Bassik MC, Brown CD, Correa A, Hall I, Im HK, Li X, Natarajan P, GTEx Consortium, Lappalainen T, Mohammadi P, Montgomery SB, Battle A. Transcriptomic signatures across human tissues identify functional rare genetic variation. 2020. In Press.
0. Gay NR, Gloudemans M, Antonio ML, **Abell NS**, Balliu B, Park Y, Martin AR, Musharoff S, Rao A, Aguet F, Barbeira B, Bonazzola R, Hormozdiari F, GTEx Consortium, Ardlie K, Brown CD, Im HK, Lappalainen T, Wen X, Montgomery SB. Impact of admixture and ancestry on eQTL analysis and GWAS colocalization in GTEx. 2020. In Press.
0. Balliu B, Durrant M, de Goede O, **Abell NS**, Li X, Liu B, Gloudemans MJ, Cook N, Smith KS, Pala M, Cucca F, Schlessinger D, Jaiswal S, Sabatti C, Lind L, Ingelsson E, Montgomery SB. [Genetic dysregulation of gene expression and splicing during a ten-year period of human aging.](https://www.ncbi.nlm.nih.gov/pubmed/31684996) Genome Biology. 2019;20(1):230.
0. Reinsborough CW<sup>&dagger;</sup>, Ipas H<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Nottingham RM<sup>&dagger;</sup>, Yao J, Devanathan SK, Shelton SB, Lambowitz AM, Xhemalce B. [BCDIN3D regulates tRNA-His 3’ fragment processing.](https://www.ncbi.nlm.nih.gov/pubmed/31329584) PLOS Genetics. 2019;15(7):e1008273.
0. Liu B<sup>&dagger;</sup>, Calton M<sup>&dagger;</sup>, **Abell NS**, Benchorin G, Gloudemans MJ, Hu J, Balliu B, Bok D, Montgomery SB, Vollrath D. [Genetic analyses of human fetal retinal pigment epithelium gene expression suggest ocular disease mechanisms.](https://www.ncbi.nlm.nih.gov/pubmed/31123710) Communications Biology. 2019;2:186.
0. Shelton SB, Shah NM<sup>&dagger;</sup>, **Abell NS**<sup>&dagger;</sup>, Devanathan S, Mercado M, Xhemalce B. [Crosstalk between the RNA methylation and histone binding activities of MePCE regulates P-TEFb activation on chromatin.](https://www.ncbi.nlm.nih.gov/pubmed/29425494) Cell Reports. 2018;22(6):1374-1383.
0. **GTEx Consortium**. [Genetic effects on gene expression across human tissues.](https://www.ncbi.nlm.nih.gov/pubmed/29022597) Nature. 2017;550(7675):204-213. [As member of the Consortium]
0. Zacharioudakis E<sup>&dagger;</sup>, Agarwal P<sup>&dagger;</sup>, Bartoli A, **Abell NS**, Kunalingam L, Bergoglio V, Xhemalce B, Miller KM, Rodriguez R. [Chromatin Regulates Genome Targeting with Cisplatin.](https://www.ncbi.nlm.nih.gov/pubmed/28474855) Angewandte Chemie. 2017;56(23):6483-6487.
0. **Abell NS**<sup>&dagger;</sup>, Mercado M<sup>&dagger;</sup>, Cañeque T, Rodriguez R, Xhemalce B. [Click Quantitative Mass Spectrometry Identifies PIWIL3 as a Mechanistic Target of RNA Interference Activator Enoxacin in Cancer Cells.](https://www.ncbi.nlm.nih.gov/pubmed/28094937) Journal of the American Chemical Society. 2017;139(4):1400-1403.
0. Tsang EK, **Abell NS**, Li X, Anaya V, Karczewski KJ, Knowles DA, Sierra RG, Smith KS, Montgomery SB. [Small RNA Sequencing in Cells and Exosomes Identifies eQTLs and 14q32 as a Region of Active Export.](https://www.ncbi.nlm.nih.gov/pubmed/27799337) G3: Genes\|Genomes\|Genetics. (Bethesda). 2017;7(1):31-39.
0. Polioudakis D, **Abell NS**, Iyer VR. [MiR-191 Regulates Primary Human Fibroblast Proliferation and Directly Targets Multiple Oncogenes.](https://www.ncbi.nlm.nih.gov/pubmed/25992613) PLoS ONE. 2015;10(5):e0126535.
0. Polioudakis D, **Abell NS**, Iyer VR. [miR-503 represses human cell proliferation and directly targets the oncogene DDHD2 by non-canonical target pairing.](https://www.ncbi.nlm.nih.gov/pubmed/25653011) BMC Genomics. 2015;16:40.
0. Polioudakis D, Bhinge AA, Killion PJ, Lee BK, **Abell NS**, Iyer VR. [A Myc-microRNA network promotes exit from quiescence by suppressing the interferon response and cell-cycle arrest genes.](https://www.ncbi.nlm.nih.gov/pubmed/23303785) Nucleic Acids Research. 2013;41(4):2239-54.

## References

Dr. Stephen Montgomery - Stanford University School of Medicine

Dr. Blerta Xhemalce - University of Texas at Austin

Dr. Vishwanath Iyer - University of Texas at Austin





